src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1204,6 +1210,115 @@
         az acr webhook update -n MyWebhook -r MyRegistry --status disabled
 """
 
+# region connected-registry
+helps['acr connected-registry'] = """
+type: group
+short-summary: Manage Edge/On-Prem Container Registries with Azure Container Registries.
```

Comment:
Are we supposed to use Edge/On-Prem? How about 

Manage connected registry resources with Azure Container Registries?